### PR TITLE
docs: update TRG 8.05 - Dependabot

### PR DESCRIPTION
**Pull request by RoKrish14**
## Description
This PR moves the existing TRG2.6 to TRG-8.05 as this is more relevant to the topic of security.

**NOTE: Please do not merge this PR until the August release**

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Review by matbmoser — APPROVED** [review]
lgtm

**Comment by RoKrish14** [comment]
**NOTE: Please do not merge this PR until the August release**

**Comment by tom-rm-meyer-ISST** [comment]
@eclipse-tractusx/automotive-tractusx-committers please review

File: docs/release/trg-8/trg-8-05.md [hunk 1]
```diff
@@ -1,9 +1,10 @@
 ---
-title: TRG 2.06 - Dependabot
+title: TRG 8.05 - Dependabot
 ---
 
 | Status | Created      | Post-History    |
 |--------|--------------|-----------------|
+| Update | 19-July-2024 | Moved the content from trg-2-6.md|
```
Comment (ndr-brt) [inline docs/release/trg-8/trg-8-05.md:7]: please keep the table formatting consistent

**Comment by evegufy** [comment]
@RoKrish14 are you still integrating review comments?

File: docs/release/trg-8/trg-8-05.md (hunk 1 above)
Comment (RoKrish14) [reply docs/release/trg-8/trg-8-05.md]: HI @ndr-brt: I have made the necessay corrections. Please verify.

**Review by ndr-brt — APPROVED** [review]

**Comment by RoKrish14** [comment]
> @RoKrish14 are you still integrating review comments?

Yes, and they are done :)

**Review by tom-rm-meyer-ISST — APPROVED** [review]
LGTM

**Pull request merged**